Rakefile

```diff
@@ -1,33 +1,18 @@
 namespace :build do
   desc 'Builds the Lottie example app'
   namespace :example do
-    task all: ['iOS:simulator', 'iOS:device', 'macOS:arm64', 'macOS:x86_64', 'macCatalyst:arm64', 'macCatalyst:x86_64']
```

Comment:
Had to remove some of the CI jobs added in #9. I expected that job to fail because of the Mac Catalyst issue, but didn't notice that several of the other jobs were also failing:
 - We can't build for iOS devices in CI because it requires a code signing identity
 - We can't build for arm64 macOS in CI because the CI machines are x86_64